docs/howto/openstack/magnum/kubectl.md

```diff
@@ -89,3 +89,31 @@ kube-system   magnum-metrics-server-79556d6999-xdlpm       1/1     Running   0
 kube-system   npd-5p6gk                                    1/1     Running   0          165m
 kube-system   openstack-cloud-controller-manager-44rz9     1/1     Running   0          167m
 ```
+
+## Defining a default storage class
+
+An OpenStack Magnum-managed cluster does not automatically define a default [storage class](https://kubernetes.io/docs/concepts/storage/storage-classes/) for [dynamic volume provisioning](https://kubernetes.io/docs/concepts/storage/dynamic-provisioning/).
+You should define it immediately upon cluster creation.
```

Comment:
Following on the previous sentence, I would change that one into the following:

```suggestion
So after creating your new cluster, you should define a storage class immediately.
```

(The way I see it ---and I may be wrong on the percentage---, leaving the original sentence would have 1,27% of the readers believe that the whole section was generated by an AI.)

Reply:
Erm, no. That doesn't sound like an improvement to me. This does, tough:
```suggestion
You should define one immediately upon cluster creation.
```